src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -858,6 +858,173 @@ public LazyBlockStatement(Lazy<ImmutableArray<IOperation>> statements, Immutable
     }
 
     /// <summary>
+    /// Represents a sequence of expressions.
+    /// </summary>
+    internal abstract partial class BaseSequenceExpression : Operation, ISequenceExpression
```

Comment:
I see generated code here, but I don't see any changes to a generate base.

Reply:
@agocke even though file is .generated.cs, it is a hand crafted file. https://github.com/dotnet/roslyn/issues/20204 tracks writing an operation generator similar to the syntax and bound node generators.

Reply:
@mavasani This is very confusing. Can we please rename it to not included `generated` in the file name?